libensemble/gen_funcs/ytopt_gen_xsbench.py

```diff
@@ -0,0 +1,57 @@
+"""
```

Comment:
This can probably be deleted, right? Its very similar to `ytopt_asktell.py` except for missing the csv-writing section

Reply:
I have asked Xingfu about which one he would like to keep